src/features/student/user/components/composites/UpdateProfileDrawer/use-update-profile-form.test.ts

```diff
@@ -148,13 +148,15 @@ BddTest().given('a useUpdateProfileForm composable', () => {
     BddTest().then('it should return validation errors', async () => {
       result.form.setFieldValue('firstname', '')
       result.form.setFieldValue('lastname', '')
+      result.form.setFieldValue('bio', Array.from({ length: 401 }).fill('a').join(''))
```

Comment:
```suggestion
      result.form.setFieldValue('bio', Array.from({ length: BIOGRAPHY_MAX_LENGTH + 1 }).fill('a').join(''))
```

Comme ça si cette constante change, t'auras pas à mettre à jour ton test ;)